### PR TITLE
Add CoP membership to usa-cards

### PR DESCRIPTION
**Pull request by ch-tyson**
## Summary

Adds membership count to usa-cards in featured communities [[preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/tc-CoP-membership/)] and communities page [[preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/tc-CoP-membership/communities/)]. 
## Preview

above

## Solution

Provide a summary of the solution this PR offers.

## How To Test

1. see if both pages look good!

### CODE REVIEW

**Comment by ToniBonittoGSA** [comment]
Hi @ch-tyson 

[Homepage preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/tc-CoP-membership/) - looks good

[Communities page preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/tc-CoP-membership/) - alignment is off

**Comment by nick-mon1** [comment]
@ch-tyson Due to some testing by @RileySeaburg there are two un-needed `yaml` files in your PR, please remove.

**Review by nick-mon1 — CHANGES_REQUESTED** [review]
Overall, this looks good @ch-tyson.
Added some minor suggestions and improvements.

### What I Tested

- a11y
- responsive layout
- code styles and markup

File: themes/digital.gov/src/scss/new/home/_communities.scss [hunk 1]
```diff
@@ -121,6 +130,11 @@ $communities-of-practice: (
       background-color: transparent;
       max-width: 4rem;
     }
+
+    .usa-card__members {
+      padding-left: units(3);
+      text-align: left;
+    }
   }
 }
```
[the remark this answers is not in the document]
Comment (ch-tyson) [reply themes/digital.gov/src/scss/new/home/_communities.scss:140]: Went ahead and deleted it!

**Pull request merged**